src/aspire/utils/rotation.py

```diff
@@ -265,6 +265,20 @@ def about_axis(axis, angles, dtype=np.float32):
         matrix = rotation.as_matrix().astype(dtype)
         return Rotation(matrix)
 
+    @staticmethod
+    def from_rotvec(vec, dtype=np.float32):
```

Comment:
Which way is the rotation 😇 ?

How to I rotate the opposite way? Is that not possible?

Reply:
Nice. Thanks!